### PR TITLE
Correct pilot autoscalemax to 5

### PR DESCRIPTION
**Pull request by linsun**
and a few other minor changes

### CODE REVIEW

File: install/kubernetes/helm/istio/values-istio-gateways.yaml [hunk 1]
```diff
@@ -13,7 +13,8 @@ global:
   # configmap in those.
   omitSidecarInjectorConfigMap: true
 
-  # istio control plane namespace
+  # Istio control plane namespace: modify this if you installed the
+  # control plane in a different namespace than istio-system.
```
Comment (costinm) [inline install/kubernetes/helm/istio/values-istio-gateways.yaml]: I don't remember why this is needed - can you add an explanation ? Normally we get this from Release.Namespace.

Comment (linsun) [reply install/kubernetes/helm/istio/values-istio-gateways.yaml]: Oh, this is needed to tell the 2nd gateway where the control plane is.  I'll clarify it.

**Comment by hklai** [comment]
@linsun it looks like this is not merged to master. Is this needed?

**Comment by linsun** [comment]
@hklai - seems it was merged in #8074 by @rshriram ?

Yes, would be good to have it in master.

**Comment by hklai** [comment]
#8074  was apparently closed and not merged....

**Pull request merged**